test/test_emulation/__init__.py

```diff
@@ -90,20 +90,17 @@ def udev_rules(self):
     @pytest.fixture(autouse=True, scope='session')
     def server(self):
         p = None
-        try:
-            stdout = open('ratbag-emu-log-stdout.txt', 'w')
-            stderr = open('ratbag-emu-log-stderr.txt', 'w')
-
-            p = subprocess.Popen(['/usr/bin/env', 'python3', '-m', 'ratbag_emu'], stdout=stdout, stderr=stderr)
-            sleep(2)
-            yield
-        finally:
-            if p:
-                p.kill()
-            if stdout:
-                stdout.close()
-            if stderr:
-                stderr.close()
+        logs = f'ratbag-emu-log-{strftime("%Y-%m-%d_%H-%M")}'
```

Comment:
use `%H:%M` because that's what we'd expect for time (well, at least the people who can handle 24h notation ;)